packages/instrumentor/plugins/functionHooks.test.ts

```diff
@@ -20,13 +20,20 @@
 import { instrumentAndEvalWith } from "./testhelpers";
 import { functionHooks } from "./functionHooks";
 import * as hooking from "@jazzer.js/hooking";
+import { Hook } from "@jazzer.js/hooking";
 
 const expectInstrumentationEval = instrumentAndEvalWith(
 	functionHooks("pkg/lib/a")
 );
 
 registerHookManagerGlobally();
 
+function expectHook(idx: number, hookCallMap: Map<number, [number, Hook]>) {
```

Comment:
Can you also add tests that show that the `hookTracker` fills the sets `applied`, `available`, and `notApplied` with expected hook names after instrumentation is done?